android/app/src/main/java/app/organicmaps/sdk/OrganicMaps.java

```diff
@@ -160,7 +159,8 @@ private void initNativePlatform() throws IOException
     createPlatformDirectories(writablePath, privatePath, tempPath);
 
     nativeInitPlatform(mContext, apkPath, writablePath, privatePath, tempPath, app.organicmaps.BuildConfig.FLAVOR,
-                       app.organicmaps.BuildConfig.BUILD_TYPE, UiUtils.isTablet(mContext));
+                       app.organicmaps.BuildConfig.BUILD_TYPE,
+                       mContext.getResources().getBoolean(app.organicmaps.sdk.R.bool.isTablet));
```

Comment:
Old code used isTabletLayout, why was it changed? What's the whole logic now with the tablet?

Reply:
:(
Need to fix that, thanks!